corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -286,6 +287,21 @@
                             hqImport('export/js/utils').capitalize(exportDownloadService.exportType), "Saved");
             hqImport('analytix/js/kissmetrix').track.event("Clicked Download button");
         };
+
+        $scope.sendEmailUponCompletion = function () {
+            setTimeout(function () {
+                if ($scope.download_id !== undefined) {
+                    $.ajax({
+                        method: 'POST',
+                        dataType: 'json',
+                        url: '../../../../../add_export_email_request/',
```

Comment:
There's infrastructure for providing URLs to JavaScript without needing to hard code them. Check out https://github.com/dimagi/js-guide/blob/master/integration-patterns.md#django-urls

Reply:
+1 to this. @xbryanc I know I showed you how to do this so you could get over the hump, but definitely worth fixing to have this follow that practice. (Keeps links from breaking if we change URLs, etc., but also is just standard practice that helps others follow the code.)